### PR TITLE
Clean up model input names for consistency

### PR DESCRIPTION
**Pull request by mattdangerw**
This proposes a few changes to the naming our our model and layer inputs
and outputs.

1) Rename `input_ids` -> `token_ids` for bert/roberta.
   Everything is an "input", including the segment id input, so I don't
   think input is a helpful naming prefix in this case.
2) Rename `input_mask`  -> `padding_mask` for bert/roberta.
   This matches the name of the variable for the transformer
   encoder/decoder argument.
3) Rename `tokens` -> `token_ids` for MLMMaskGenerator.
   This layer only operates in id space, so I think token_ids is more
   descriptive and consistent with above.

### CODE REVIEW

**Comment by jbischof** [comment]
Looks good in general but I'm confused by the `padding_mask` label. Aren't we also masking the holdout words in pretraining and not just the padding?

Does the `examples/bert/` code still work? I see it still uses `input_ids` (here)[https://github.com/keras-team/keras-nlp/blob/master/examples/bert/bert_train.py#L316].

Finally, one other naming inconsistency you could mop up here [sine_position_encoding](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/sine_position_encoding.py#L66).

**Review by jbischof — APPROVED** [review]
LGTM! Added some comments for consideration but trust your judgment.

**Comment by mattdangerw** [comment]
@jbischof Thanks! And good call I will rename in `examples/bert` to match.

Re `padding_mask` and the MLM masked words, that's actually a different mask I think. There's actually a `[PAD]` token and a `[MASK]` token. During pretraining, `[PAD]` will be used only to pad sequences, and `[MASK]` will be used for the MLM labels.

So I do think it is accurate to use `padding_mask`. Other options are `mask` or `token_mask`. I am actually fine with any of those. I was just choosing `padding_mask` because that the name we chose for our TransformerEncoder/TransformerDecoder https://keras.io/api/keras_nlp/layers/transformer_encoder/#call-method.

**Review by fchollet — APPROVED** [review]
LGTM
